apps/learneth/src/actions/index.ts

```diff
@@ -0,0 +1,371 @@
+import {toast} from 'react-toastify'
+import axios from 'axios'
+import groupBy from 'lodash/groupBy'
+import pick from 'lodash/pick'
+import remixClient from '../remix-client'
+import {router} from '../App'
+
+let dispatch, state
+
+// const apiUrl = 'http://localhost:3001';
+const apiUrl = 'https://static.220.14.12.49.clients.your-server.de:3000'
```

Comment:
whose address is that?

Reply:
It's the backend service address that I got from the old learneth plugin. @yann300 

https://github.com/bunsenstraat/remix-learneth-plugin/blob/master/src/environments/environment.prod.ts#L4

Reply:
we will change that now that the plugin is internal. I will update it later.

Reply:
could this PR be merged in the meantime?